WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/SystemStatusReport/SystemStatusReportViewModel.swift

```diff
@@ -41,8 +41,187 @@ final class SystemStatusReportViewModel: ObservableObject {
+            }
+        }
+
+        // Post Type Counts
+        if systemStatus.postTypeCounts.isNotEmpty {
+            lines.append("\n### Post Type Counts ###\n")
+            for postTypeCount in systemStatus.postTypeCounts {
+                lines.append("\(postTypeCount.type): \(postTypeCount.count)")
+            }
+        }
+
+        // Security
+        if let security = systemStatus.security {
+            lines.append(contentsOf: [
+                "\n### Security ###\n",
+                "Secure connection (HTTPS): \(security.secureConnection.stringRepresentable)",
+                "Hide errors from visitors: \(security.hideErrors.stringRepresentable)"
+            ])
+        }
+
+        // Active plugins
+        if systemStatus.activePlugins.isNotEmpty {
+            lines.append("\n### Active Plugins (\(systemStatus.activePlugins.count)) ###\n")
+            for plugin in systemStatus.activePlugins {
+                lines.append("\(plugin.name): by \(plugin.authorName) - \(plugin.version)")
+            }
+        }
+
+        // Inactive plugins
+        if systemStatus.inactivePlugins.isNotEmpty {
+            lines.append("\n### Inactive Plugins (\(systemStatus.inactivePlugins.count)) ###\n")
+            for plugin in systemStatus.inactivePlugins {
+                lines.append("\(plugin.name): by \(plugin.authorName) - \(plugin.version)")
+            }
+        }
+
+        // Dropin plugins
+        if systemStatus.dropinPlugins.isNotEmpty {
+            lines.append("\n### Dropin Plugins (\(systemStatus.dropinPlugins.count)) ###\n")
+            for plugin in systemStatus.dropinPlugins {
+                lines.append("\(plugin.plugin): \(plugin.name)")
+            }
+        }
+
+        // Must Use plugins
+        if systemStatus.mustUsePlugins.isNotEmpty {
+            lines.append("\n### Must Use Plugins (\(systemStatus.mustUsePlugins.count)) ###\n")
+            for plugin in systemStatus.mustUsePlugins {
+                lines.append("\(plugin.plugin): \(plugin.name)")
+            }
+        }
+
+        // Settings
+        if let settings = systemStatus.settings {
+            lines.append(contentsOf: [
+                "\n### Settings ###\n",
+                "API Enabled: \(settings.apiEnabled.stringRepresentable)",
+                "Force SSL: \(settings.forceSSL.stringRepresentable)",
+                "Currency: \(settings.currency)",
+                "Currency Position: \(settings.currencyPosition)",
+                "Thousand Separator: \(settings.thousandSeparator)",
+                "Decimal Separator: \(settings.decimalSeparator)",
+                "Number of Decimals: \(settings.numberOfDecimals)"
+            ])
+
+            for (index, content) in settings.taxonomies.enumerated() {
+                if index == 0 {
+                    lines.append("\nTaxonomies: Product Types: \(content.key) (\(content.value))")
```

Comment:
nit: just wondering, why not show the first line of content after the title line?

Reply:
I wanted to copy the style in Core to every detail. It really makes the logic more complicated, but I'm not sure about making my own formatting so 😛